### ISSUE TITLE
Contract to Update BC Website's Jekyll Theme

### ISSUE THREAD

**Issue by ChristopherA** [BlockchainCommons/www.blockchaincommons.com#4]
We'd like to identify and contract a web developer for a number of updates to the static website software use use for Blockchain Commons https://www.BlockchainCommons.com.

We need a web developer that specializes in Jekyll, and in particular is comfortable with the requirements of remote-theme functional Jekyll themes. There were some web developers listed on [Fivrr](https://fivrr.com) that looked like they might be good for custom Jekyll themes, but other outsourcing sites may be better.

For now, we want our current pages, and especially the blog content on the pages, to moves over, but we want to stay within general features and look & feel of the current theme ["Minimal Mistakes - So Simple"](https://github.com/mmistakes-so-simple), but make changes to our [fork](https://github.com/BlockchainCommons/so-simple-theme) of that theme. 

Unfortunately, the "So Simple" theme’s simpleness leaves out several common Jekyll template features that we want to have. 

Use this issue in our website repo's [issues section](https://github.com/BlockchainCommons/www.blockchaincommons.com/issues) to track issues for each of our desired features below:

* Issue #5
* Issue #6
* Issue #7
* Issue #8
* Issue #9

Note that all of these features exist in other Jekyll themes, nothing is particular unusual. But ideally once we have this working, we can ask for other features or graphical updates.

**Comment by shannona** [comment]
#5 and #8 remain open at this point.

**Comment by ChristopherA** [comment]
Can you summarize these (and any other) changes you’d like to see I our Jekyll based sights as a community project, and point to it in discussions as a possible internship project?

**Comment by shannona** [comment]
this was largely closed out, with the exception of #8, which I'll leave open separately.
